core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy

```diff
@@ -42,12 +42,10 @@ f.section(title: _("Administrative monitors configuration")) {
                                 checked: am.enabled,
                                 json: am.id)
                     }
-                    tr() {
-                        td(colspan: "2")
-                        td(class: "setting-description") {
+                    div(class: "tr") {
```

Comment:
why use a `tr` class, seems confusing (can the class not be renamed?)

Reply:
`tr` was from my original migration. As part of "get it working" before "get it pretty".